lib/expartisan.ex

```diff
@@ -82,6 +87,16 @@ defmodule ExPartisan do
 
   @doc """
   Disconnects the specified `node`.
+
+  ## Examples
+
+      iex(foo@127.0.0.1)> ExPartisan.members
+      {:ok, [:"foo@127.0.0.1", :"bar@127.0.0.1"]}
+      iex(foo@127.0.0.1)> ExPartisan.leave(bar)
+      :ok
+      iex(foo@127.0.0.1)> ExPartisan.members
```

Comment:
Should we make members return a map?  That might be a good change across the board.

Reply:
After looking into this, this is a bigger change than initially thought.